indices.py

```diff
@@ -8,8 +8,15 @@
 # Figure out clue indices and update grid.
 #
 
+import logging
 import tkinter as tk
 
+LOGGER_FORMAT = "%(filename)s:%(lineno)s %(funcName)s: %(message)s"
+#LOGGER_LEVEL = logging.INFO
+LOGGER_LEVEL = logging.DEBUG
+logging.basicConfig( format=LOGGER_FORMAT, level=LOGGER_LEVEL)
+logger = logging.getLogger(__name__)
+
```

Comment:
Rather than add a central logger config file I re-add the same logic to each file, which means debugging must be manually enabled per-file.  There is probably a smarter way to use the **logging** module but I figured this would be a good start.